integration_tests/tests/consistency_pardot__lists.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{% set exclude_cols = [] + var('consistency_test_exclude_metrics', []) %}
```

Comment:
Can remove the first [] if nothing is being brought in by default.

```suggestion
{% set exclude_cols = var('consistency_test_exclude_metrics', []) %}
```